chainer/optimizers/data_parallel.py

```diff
@@ -0,0 +1,148 @@
+import chainer
+import chainer.optimizer_hooks
+
```

Comment:
Should probably import `optimizer` similar to how it is done here


https://github.com/justusschock/chainer/blob/f8c9f1507de5828422131a69e1e866f330f3fd86/chainer/optimizers/rmsprop.py#L5

This removed import error atleast from this file.